File: 01-05-2026/name_initials.js
function getInitials(name) {
  return name
    .split(" ")
    .map((word) => word[0].toUpperCase())
    .join(".")
    .concat(".");
}

console.log(getInitials("Tommy Millwood")); // "T.M."
console.log(getInitials("Savanna Puddlesplash")); // "S.P."
console.log(getInitials("Frances Cowell Conrad")); // "F.C.C."
console.log(getInitials("Dragon")); // "D."
console.log(getInitials("Dorothy Vera Clump Haverstock Norris")); // "D.V.C.H.N."

File: 01-05-2026/name_initials.py
def get_initials(name):
    name = name.split()
    name = [n[0] for n in name]
    
    return ".".join(name) + "."

print(get_initials("Tommy Millwood")); # "T.M."
print(get_initials("Savanna Puddlesplash")); # "S.P."
print(get_initials("Frances Cowell Conrad")); # "F.C.C."
print(get_initials("Dragon")); # "D."
print(get_initials("Dorothy Vera Clump Haverstock Norris")); # "D.V.C.H.N."
